pkg/converter/tool/builder.go

```diff
@@ -75,16 +79,17 @@ func Pack(option PackOption) error {
 		args = append(args, "--compressor", option.Compressor)
 	}
 	args = append(args, option.SourcePath)
-
+	ctx, cancel := context.WithTimeout(context.Background(), maxTimeOut)
+	defer cancel()
 	logrus.Debugf("\tCommand: %s %s", option.BuilderPath, strings.Join(args[:], " "))
 
-	cmd := exec.Command(option.BuilderPath, args...)
+	cmd := exec.CommandContext(ctx, option.BuilderPath, args...)
 	cmd.Stdout = logger.Writer()
 	cmd.Stderr = logger.Writer()
 	cmd.Stdin = strings.NewReader(option.PrefetchPatterns)
 
 	if err := cmd.Run(); err != nil {
-		logrus.WithError(err).Errorf("fail to run %v %+v", option.BuilderPath, args)
+		logrus.WithError(err).Errorf("fail to run %v %+v, possibly due to timeout %v", option.BuilderPath, args, maxTimeOut)
```

Comment:
Should check the error string `signal: killed` and then give `possibly due to timeout`.